packages/dashboards/package.json

```diff
@@ -74,6 +74,7 @@
     "ember-tooltips": "^2.9.2",
     "ember-truth-helpers": "^2.0.0",
     "ember-uuid": "^1.0.0",
+    "liquid-fire": "^0.29.2",
```

Comment:
we pull in `navi-reports` into this. Since `liquid-fire` is in the navi-reports blueprint is this still required here?